resources/unity/PlatformSH/files/solr_config/solrcore.properties

```diff
@@ -1,16 +1,13 @@
 solr.replication.master=false
 solr.replication.slave=false
 solr.replication.pollInterval=00:00:60
-solr.replication.masterUrl=http://solr.internal:8080/solr
-solr.replication.confFiles=schema.xml,schema_extra_types.xml,schema_extra_fields.xml,elevate.xml,stopwords_en.txt,synonyms_en.txt,protwords_en.txt,accents_en.txt,stopwords_und.txt,synonyms_und.txt,protwords_und.txt,accents_und.txt
+solr.replication.masterUrl=http://localhost:8983/solr
```

Comment:
Double check this but I'm pretty sure PSH uses 8080 for the Solr port.

Reply:
Good spot ! Have now corrected that